app/models/publisher.rb

```diff
@@ -37,6 +58,21 @@ def save_catching_gds_api_errors(&block)
     end
   end
 
+  class DiscardDraftResponse
+    def initialize(opts = {})
+      @success = opts.fetch(:success)
+      @errors = opts.fetch(:errors, [])
+    end
+
+    def success?
+      @success
+    end
+
+    def errors
+      @errors
+    end
+  end
```

Comment:
We could rename `PublicationResponse` to be applicable to both. The interface for the two are nearly identical.